src/pages/publication/SubmissionPanel.tsx

```diff
@@ -64,20 +64,20 @@ const SubmissionPanel: FC<SubmissionPanelProps> = ({ isSaving, savePublication }
 
   const onClickPublish = async () => {
     setIsPublishing(true);
-    if (dirty) {
-      await savePublication(values);
-    }
-    const publishedPublication = await publishPublication(values.identifier);
-    if (publishedPublication?.error) {
-      setIsPublishing(false);
-      dispatch(setNotification(publishedPublication.error, NotificationVariant.Error));
-    } else if (publishedPublication?.info) {
-      dispatch(setNotification(publishedPublication.info, NotificationVariant.Info));
-      setTimeout(() => {
+    const publicationIsUpdated = dirty ? await savePublication() : true;
+    if (publicationIsUpdated) {
+      const publishedPublication = await publishPublication(values.identifier);
+      if (publishedPublication?.error) {
+        setIsPublishing(false);
+        dispatch(setNotification(publishedPublication.error, NotificationVariant.Error));
+      } else if (publishedPublication?.info) {
+        dispatch(setNotification(publishedPublication.info, NotificationVariant.Info));
+        setTimeout(() => {
+          history.push(`/registration/${values.identifier}/public`);
+        }, NAVIGATE_TO_PUBLIC_PUBLICATION_DURATION);
```

Comment:
vet du hvorfor er den en timeout her btw @sarahse ?
ser ut som vi bare kan fjerne den og kanskje droppe info om at "publikasjonen publiseres..." også, da Publish-knappen uansett har loading state

Reply:
ja, jeg lurer på om det stod i en feature? eller så var det fordi det tok så lang tid å publisere en publikasjon at vi måtte ha noe "i mellomtiden"

Reply:
hm, lar det bare ligge for nå...